templates/game.html.twig

```diff
@@ -1,28 +1,14 @@
 {% extends "base.html.twig" %}
 
 {% block body %}
+<div>
+Tour du joueur : {{ game.currentPlayer.name }}
```

Comment:
Please use translation and english first

Reply:
I will add i18n later if you are ok ?

Reply:
It'll takes you about 10 minutes for all the app